twinPicks/jsonld-contexts/twinPicks-compound.jsonld

```diff
@@ -1,11 +1,17 @@
 {
   "@context": [
-    "https://easy-global-market.github.io/ngsild-api-data-models/dcat/jsonld-contexts/dcat-compound.jsonld",
+    "https://easy-global-market.github.io/c2jn-data-models/jsonld-contexts/datacenter-equipments.jsonld",
```

Comment:
in the future, this one and `c2jn-smart-city-compound.jsonld` shoud be moved away from the core TP (but that may be difficult to do right now)